Makefile

```diff
@@ -14,4 +14,4 @@ push:
 	docker push $(REPO)
 
 ropsten:
-	docker run -p 18545:8545 -p 18546:8546 $(REPO):latest --chain ropsten
+	docker run -p 18545:8545 -p 18546:8546 $(REPO):latest --config testnet.toml
```

Comment:
The reason why I didn't initially use `testnet.toml` has paths specific to kubernetes.

Notice the `base_path = "/data/ethereum"`. This is different to `miner.toml` which is what `docker run` uses by default: https://github.com/smartcontractkit/devnet/blob/master/Dockerfile#L5

I would move `testnet.toml` into `./k8s/.` and then make another ropsten.toml specific for local use.